lang/ja.json

```diff
@@ -436,7 +436,7 @@
       "artCredit": "Art by {ARTIST}"
```

Comment:
Translate this too

Reply:
Perhaps it should leave it as is.
On the other hand, I can't think of a translation.
○○ by ○○
is often used in Japan